website/pages/explore/dir-content/kanban.asciidoc

```diff
@@ -0,0 +1,28 @@
+= KANBAN
+
+[.directory]
+== Kanban
+
+Generally devonfw and Kanban can be used together.
```

Comment:
Better: "Kanban as a process is completely valid to be applied. devonfw does not make any prescriptions in regards to the process."